src/common/task_manager/tests.rs

```diff
@@ -14,20 +14,27 @@ use crate::common::task_manager::manager::SpawnError;
 use crate::common::task_manager::task::{TaskName, EDGES};
 use crate::common::task_manager::{wait_for_shutdown, TASK_MANAGER};
 
-#[tokio::test]
-async fn test_dependency_graph() {
+#[test]
```

Comment:
Can you revert the changes in test to see what happend?

Reply:
tokio::test will start a new tokio runtime and drop the temp task manager in current runtime, it is not allowed in async context:
```bash
.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.37.0/src/runtime/blocking/shutdown.rs:51:21:
Cannot drop a runtime in a context where blocking is not allowed. This happens when a runtime is dropped from within an asynchronous context.
```